docs/technical_reference/flowsheets/seawater_RO_desalination.rst

```diff
@@ -0,0 +1,200 @@
+   "Energy recovery device permeate side pressure", "101325", ":math:`\text{Pa}`", "``m.fs.desalination.ERD``"
+   
+   **Post-Treatment**
+   "Anti-scalant dose", "5", ":math:`\text{mg/L}`", "``m.fs.posttreatment.anti_scalant_addition``"
+   "Lime dose", "2.3", ":math:`\text{mg/L}`", "``m.fs.posttreatment.lime_addition``"
+   "Storage tank 2 storage time", "1", ":math:`\text{hr}`", "``m.fs.posttreatment.storage_tank_2``"
+   "Storage tank 3 storage time", "1", ":math:`\text{hr}`", "``m.fs.posttreatment.storage_tank_3``"
+   "UV/AOP reduction equivalent dose", "350", ":math:`\text{mJ/}\text{cm}^2`", "``m.fs.posttreatment.uv_aop``"
+   "UV/AOP UV transmittance", "0.95", ":math:`\text{dimensionless}`", "``m.fs.posttreatment.uv_aop``"
+
+
+
+Implementation
+--------------
+
+The demonstration file itself contains several core functions that are used to build, specify, initialize, and solve the model, as well as
+some helper functions that group these core functions together for convenience. Building and solving the flowsheet proceeds in six steps:
+
+1. Creating and instantiating the model using ``build()``:
+
+    This function will create the core components and structure of the flowsheet. The keyword argument ``erd_type`` dictates the type
+    of energy recovery device to be used, with options ``pressure_exchanger`` or ``pump_as_turbine``.
+    First, the ``FlowsheetBlock``, ``Database``, property models, are created. The zero order property models require the user
+    to provide a ``solute_list`` (e.g., TDS and TSS), while the seawater property model is pre-populated with TDS as the only solute.
+    Separate ``Block`` are created to contain all the unit models required to model the pretreatment, desalination, and post-treatment
+    parts of the treatment train:
+
+        * Pre-treatment (``m.fs.pretreatment``): comprised entirely of zero order models, this block contains the intake, chemical addition, media, and cartridge filtration unit models.
+        * Desalination (``m.fs.desalination``): contains all the unit models needed to represent the pumping, reverse osmosis (RO), and energy recovery device (ERD) processes.
+        * Post-treatment (``m.fs.posttreatment``): includes post desalination disinfection, remineralization, and storage unit models.
+
+    Outside of these, the there is a feed, distribution, landfill, and disposal block that are placed directly on the flowsheet.
+    ``Translator`` blocks are added with appropriate constraints and ``Arc`` are used to connect the unit processes in the proper order.
+    Finally, default scaling factors are set and scaling factors are calculated for all variables.
+
+2. Specify the operating conditions with ``set_operating_conditions()``:
+
+    This function begins by specifying the inlet conditions as outlined in Table 2. Then, starting with the ``pretreatment`` block, the operating 
+    conditions for each unit model are set according to Table 3.
```

Comment:
The tables technically aren't labelled as Table 2 and Table 3, but perhaps it's obvious enough which tables are being referred to here. Alternatively, you could just refer them to the "Flowsheet Specifications" section as mentioned in one of the above comments.